src/test/unit/window-console.test.js

```diff
@@ -0,0 +1,38 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+import {
+  addDataToWindowObject,
+  logFriendlyPreamble,
+} from '../../utils/window-console';
+import { storeWithSimpleProfile } from '../fixtures/stores';
+
+describe('console-accessible values on the window object', function() {
+  // Coerce the window into a generic object, as these values aren't defined
+  // in the flow type definition.
+
+  it('does not have the values initially', function() {
+    expect((window: any).profile).toBeUndefined();
+    expect((window: any).filteredProfile).toBeUndefined();
+    expect((window: any).callTree).toBeUndefined();
+  });
+
+  it('adds values to the console', function() {
+    const store = storeWithSimpleProfile();
+    addDataToWindowObject(store.getState);
+    expect((window: any).profile).toBeTruthy();
```

Comment:
I felt it would be simpler to check that these were present, rather than checking for strict equality from the results of a selector.

Reply:
see my comment below about passing `window` to `addDataToWindowObject`. Then you can do here:
```javascript
const fakeWindow = {};
addDataToWindowObject(fakeWindow, store.getState);
expect(fakeWindow).toEqual({
  profile: expect.anything(),
  filteredThread: expect.anything(),
  callTree: expect.anything(),
});
```

I think this is cleaner.

Reply:
The getters make `expect(fakeWindow).toEqual({ ... })` not work. I actually like modifying the global window, as it's closest to testing the real desired behavior. Although I see now that I didn't clean up after myself, so I'm going to do that.

Reply:
Changed my mind again, I'm going to use the `fakeWindow` idea, since you can't delete or undo `Object.defineProperty`.